src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/PropertyListedHhProcessor.java

```diff
@@ -48,6 +48,7 @@ public UUID process(OutcomeSuperSetDto outcome, UUID caseIdHolder, String type)
         ADDRESS_TYPE, "HH");
 
     GatewayCache plCache = gatewayCacheService.getById(String.valueOf(caseId));
+    cacheData(outcome, newCaseId);
```

Comment:
When we `gatewayEventManager.triggerEvent(String.valueOf(caseId), OUTCOME_SENT,` We need to log the newCaseId as well